requirements.d/dev.txt

```diff
@@ -3,6 +3,7 @@ coverage
 flake8
 macholib
 nox
+peewee
```

Comment:
Since peewee is a production dep it shouldn't be needed to specify it here as well.